core/src/main/java/io/smallrye/openapi/runtime/util/TypeUtil.java

```diff
@@ -110,7 +110,7 @@ public class TypeUtil {
     public static final IndexView jdkIndex;
     private static final Set<DotName> wrapperTypes = new HashSet<>();
 
-    // https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.0.md#dataTypeFormat
+    // https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#data-type-format
```

Comment:
This one didn't change in the spec ¯\_(ツ)_/¯
```suggestion
    // https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#dataTypeFormat
```